index.js

```diff
@@ -16,12 +16,52 @@ mongoose
   })
   .then(self => {
     console.log(`Connected to the database: "${self.connection.name}"`);
-    // Before adding any documents to the database, let's delete all previous entries
     return self.connection.dropDatabase();
   })
-  .then(() => {
-    // Run your code here, after you have insured that the connection was made
+  .then(() => { // Iteration 2 and 3 
+    console.log('Database has been dropped')
+    Recipe.create({ title: "Pizza", cuisine: "Italian" })
+      //createRecipe
+      .then((response) => {
+        console.log(response)
+      })
+    let recipeMany = Recipe.insertMany(data)
+    recipeMany
+      .then(() => {
+        console.log('all added')
+      })
+    Promise.all([recipeMany]) //iteration 4
+      .then(() => {
+        let updatePromise = Recipe.updateOne({ title: "Rigatoni alla Genovese" }, { $set: { duration: 100 } })
+        updatePromise
+          .then(() => {
```

Comment:
the .then of this promise might be redundant because you already have a .then on the promise.all below. You can simply save the promise into a variable and then run the console.log inside the promise.all .then.